### DISCUSSION TITLE
Add a define for a LTS version

### DISCUSSION THREAD

**Discussion by ghost** [FirstGearGames/FishNet#252] [Feature Request]
Add a define like FISHNET_LTS_2 (for 2.0 LTS) so asset devs can make their assets backwards compatible easily.

**Comment by FirstGearGames** [comment]
Good idea. I'll probably just do FISHNET_2

**Comment by FirstGearGames** [comment]
I've decided against doing this for 2X but going forward version defines will be available as `FISHNET_V#`

Added in 3.5.3.
